Bukkit/src/main/java/com/plotsquared/bukkit/listener/EntityEventListener.java

```diff
@@ -323,6 +324,10 @@ public void onBigBoom(EntityExplodeEvent event) {
             }
         }
         event.setCancelled(true);
+        //Spawn Explosion Particles when enabled in settings
+        if (Settings.General.ALWAYS_SHOW_EXPLOSIONS){
```

Comment:
Put a space between the closing if condition and opening curly bracket.
Otherwise, looks good so far.

Reply:
Fixed in : [68ae02e](https://github.com/IntellectualSites/PlotSquared/pull/3436/commits/68ae02eadcc88699cd184fea559e99eca4fbf689)